src/v3/src/components/AuthenticatorButton/AuthenticatorButton.tsx

```diff
@@ -87,34 +89,42 @@ const AuthenticatorButton: UISchemaElementComponent<{
     });
   };
 
+  // eslint-disable-next-line no-nested-ternary
+  const ctaIcon = isAdditionalEnroll
+    ? <AddIcon titleAccess={ctaLabel} />
+    : languageDirection === 'rtl'
+      ? <ArrowLeftIcon titleAccess={ctaLabel} />
+      : <ArrowRightIcon titleAccess={ctaLabel} />;
```

Comment:
Any strong reason why you decided to disable `no-nested-ternary` instead of using `if-else`?

Reply:
No. Changed to if-else